test_libs/pyspec/eth2spec/test/epoch_processing/test_process_crosslinks.py

```diff
@@ -103,19 +104,24 @@ def test_single_crosslink_update_from_previous_epoch(state):
 
 @spec_state_test
 def test_double_late_crosslink(state):
+    if spec.SLOTS_PER_EPOCH < spec.SHARD_COUNT:
```

Comment:
This isn't the proper condition here. should be

```suggestion
    if get_epoch_committee_count(state, get_current_epoch(state)) < spec.SHARD_COUNT:
```
```